### DISCUSSION TITLE
guide/features/bulletin/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#355] [docs-comment]
# guide/features/bulletin/

概述 公告板是一个实时通知组件，可以实现在站点中方便的展示一些通知信息。 如左上角的公告板。 使用 主题提供了非常方便且灵活的 公告板 使用方法。你可以根据自己的需求，选择合适的方式进行配置。 配置说明 简单公告 当您仅需要配置 简单的公告板，仅包含一些简要的内容 时，可以直接使用 bulletin.content 添加内容。 还可以声明 bullet...

https://theme-plume.vuejs.press/guide/features/bulletin/

**Comment by dingyuqi** [comment]
长内容公告的示例代码中应该添加 `import path from 'node:path'`, 因为使用到了 `path.join`.
